### PR TITLE
feat(client): add dropdown button

### PR DESCRIPTION
**Pull request by philippfromme**
First iteration of a dropdown button.

### CODE REVIEW

File: client/src/app/primitives/Button.less [hunk 1]
```diff
@@ -1,54 +1,21 @@
...
-    color: white;
-    border-color: #489d12;
-
-    &:hover {
-      background: #2F8400;
-      border-color: #2F8400;
-    }
-  }
-
-  &.disabled {
```
Comment (nikku) [inline client/src/app/primitives/Button.less]: Is there a reason you replaced the locally scoped style with a style global `.disabled` declaration?

For the sake of understandability I'd keep the local style override pattern:

```less
:local(.someCls) {
  &.someModifierCls { ... }
}
```

Referring to it in code:

```
return <div className={ classNames(style.someCls, { someModifierCls } }>;
```

File: client/src/app/primitives/DropdownButton.less [hunk 2]
```diff
@@ -0,0 +1,61 @@
+:local(.DropdownButton) {
```
Comment (nikku) [inline client/src/app/primitives/DropdownButton.less]: Same comment like above applies here:

I'd create styles based on a single shared parent that is exposed via a `:local` declaration:

```less
:local(.DropdownContainer) {
   > .dropdown-button { ... }
   > .dropdown { ... }
}
```

Comment (nikku) [inline client/src/app/primitives/DropdownButton.less]: Are we able to re-use or mixin the existing `Button` styles? They already declare color, hover states and borders.

Comment (philippfromme) [reply client/src/app/primitives/DropdownButton.less]: Will fix. I'm not super familiar with the syntax yet. 😞

**Pull request merged**